### PR TITLE
Fix reports generated from templates stored in database

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@jbchouinard est-ce que je peux merger ?

File: report_aeroo/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    @http.route('/web/report', type='http', auth="user")
+    @serialize_exception
+    def index(self, action, token):
+        """Add the correct filename to the generated report.
+
+        Otherwise the filename is merely the name of the report.
+        """
+        response = super(Reports, self).index(action, token)
+
+        action_data = simplejson.loads(action)
```
Comment (joaoalf) [inline report_aeroo/controllers/main.py:28]: @dufresnedavid I think that simplejson is not on odoo's requirement.txt

File: report_aeroo/controllers/main.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            env = api.Environment(
+                cr, request.session.uid, context)
+            report_xml = env['ir.actions.report.xml'].search(
+                [('report_name', '=', action_data['report_name'])])
+
+            if report_xml.attachment:
+                response.headers['Content-Disposition'] = (
+                    "attachment; filename*=UTF-8''%s.%s" % (
+                        safe_eval(report_xml.attachment, {
+                            'object': env[report_xml.model].browse(ids[0]),
```
Comment (joaoalf) [inline report_aeroo/controllers/main.py:50]: @dufresnedavid What happens if you print severals reports at once?

File: report_aeroo/report_aeroo.py [hunk 3]
```diff
@@ -136,6 +120,11 @@ def create(self, cr, uid, ids, data, context=None):
         else:
             context = dict(context)
 
+        if len(ids) > 1:
```
Comment (joaoalf) [inline report_aeroo/report_aeroo.py:123]: @dufresnedavid Here's the answer to my last question.

**Review by joaoalf — COMMENTED** [review]
It would be better if you split this merge request. The cmis part doesn't seen related to the initial problem.

File: report_aeroo/report_aeroo.py (hunk 3 above)
Comment (ghost) [reply report_aeroo/report_aeroo.py:123]: @joaoalf On élimine volontairement la possibilité de produire un rapport pour plusieurs objets sur les rapports Aeroo. Ça ne fait pas partie du use case normal pour ce genre de rapport.

**Comment by ghost** [comment]
@joaoalf added the missing python libraries to the external dependencies.

The cmis library is unrelated. however, I tagged it as not installable in the manifest. I did not migrate this module. However, I want to keep the branches 8.0 and 10.0 synced.

**Comment by joaoalf** [comment]
:+1: lgtm

**Comment by ghost** [comment]
@joaoalf cool merci

**Pull request merged**